lib/resources/kernel_module.rb

```diff
@@ -37,19 +69,56 @@ def loaded?
       !found.nil?
     end
 
-    def version
-      if inspec.os.redhat? || inspec.os.name == 'fedora'
-        modinfo_cmd = "/sbin/modinfo -F version #{@module}"
-      else
-        modinfo_cmd = "modinfo -F version #{@module}"
-      end
+    # @note maintain logical agreement with 'disabled?' method
```

Comment:
This line and the one after it should just be removed rather than commented out.